Presscotts_Mortar_Calculator_(no_envirement)_V0.2.py

```diff
@@ -19,215 +18,225 @@
+        1950: [1252, 6, .7, 35.2, 3.2, 8.3, -7.8, .3, -.3, -4.7, 4.4],
+        2000: [1242, 6, .7, 35.1, 3.1, 8.4, -7.8, .3, -.3, -4.8, 4.5],
+        2050: [1230, 7, .7, 34.9, 3, 8.4, -7.9, .3, -.3, -4.9, 4.6],
+        2100: [1219, 7, .7, 34.8, 2.9, 8.5, -8, .3, -.3, -5, 4.8],
+        2150: [1207, 7, .7, 34.6, 2.9, 8.5, -8, .3, -.3, -5.2, 4.9],
+        2200: [1196, 8, .7, 34.5, 2.8, 8.6, -8.1, .3, -.3, -5.3, 5],
+        2250: [1184, 8, .7, 34.3, 2.7, 8.7, -8.2, .3, -.3, -5.4, 5.1],
+        2300: [1171, 9, .8, 34.1, 2.7, 8.7, -8.2, .3, -.3, -5.5, 5.2],
+        2350: [1158, 9, .8, 33.9, 2.6, 8.8, -8.3, .3, -.3, -5.7, 5.4],
+        2400: [1145, 10, .8, 33.7, 2.5, 8.8, -8.3, .3, -.3, -5.8, 5.5],
+        2450: [1132, 10, .8, 33.5, 2.5, 8.9, -8.4, .3, -.3, -5.9, 5.6],
+        2500: [1118, 11, .8, 33.3, 2.4, 8.9, -8.4, .3, -.3, -6, 5.7],
+        2550: [1103, 12, .8, 33.1, 2.3, 9, -8.5, .3, -.3, -6.1, 5.8],
+        2600: [1088, 13, .9, 32.8, 2.3, 9, -8.5, .3, -.3, -6.3, 5.9],
+        2650: [1072, 14, .9, 32.5, 2.2, 9, -8.6, .4, -.4, -6.4, 6],
+        2700: [1056, 15, .9, 32.2, 2.2, 9, -8.6, .4, -.4, -6.5, 6.1],
+        2750: [1038, 16, 1, 31.9, 2.1, 9, -8.6, .4, -.4, -6.6, 6.2],
+        2800: [1019, 18, 1, 31.6, 2, 9, -8.6, .4, -.4, -6.7, 6.4],
+        2850: [999, 20, 1.1, 31.2, 2, 9, -8.6, .4, -.4, -6.8, 6.4],
+        2900: [978, 22, 1.1, 30.7, 1.9, 9, -8.6, .4, -.4, -6.9, 6.5],
+        2950: [954, 26, 1.2, 30.3, 1.9, 9, -8.6, .4, -.4, -7, 6.6],
+        3000: [926, 31, 1.4, 29.7, 1.8, 8.9, -8.5, .4, -.4, -7.1, 6.7],
+        3050: [893, 39, 1.6, 28.9, 1.7, 8.8, -8.4, .4, -.4, -7.1, 6.8],
+        3100: [848, 54, 2, 27.9, 1.6, 8.5, -8.2, .4, -.4, -7.2, 6.8]
+    }
+
+    print('*' * 90)
     print('[INPUT]')
-    print('='*90)
-    posMortar = str (input("Enter your position (The Gridmark, 8-10 digit, no spaces or punctuation): "))
-    posTarget = str (input("Enter your targets position (The Gridmark, 8-10 digit, no spaces or punctuation): "))
+    print('=' * 90)
+    posMortar = str(
+        input(
+            "Enter your position (The Gridmark, 8-10 digit, no spaces or punctuation): "
+        ))
+    posTarget = str(
+        input(
+            "Enter your targets position (The Gridmark, 8-10 digit, no spaces or punctuation): "
+        ))
 
-    print('='*90)
+    print('=' * 90)
     print('[OUTPUT]')
-  
+
     if len(posMortar) % 2 == 1:
         print('ERROR: lenth of gridmark is odd, renter gridmarks')
         main()
-    if len(posTarget)% 2 == 1:
-        print('ERROR: lenth of gridmark is odd, renter gridmarks')    
+    if len(posTarget) % 2 == 1:
+        print('ERROR: lenth of gridmark is odd, renter gridmarks')
         main()
     '''Determining X,Y coordinates and setup for distance'''
     posMortarXY = str(posMortar)
-    mUpper = ((len (posMortarXY))/2) 
-    mUpper = int(mUpper) 
+    mUpper = ((len(posMortarXY)) / 2)
+    mUpper = int(mUpper)
     xGridM = posMortarXY[0:mUpper:1]

```

Comment:
Sorry if this is a bit of a dumb question but what does this do? (insert obligatory not a python guy excuse here)